react/src/js/icons/new-post.svg

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
```html

    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.1/css/all.css" integrity="sha384-5sAR7xN1Nv6T6+dT2mhtzEpVJvfS3NScPQTrOxhwjIuvcA67KV2R5Jz6kr4abQsz" crossorigin="anonymous">
```
then...
```html
<i className="fa fa-icon-name" />
```

It's better to import a CDN with icons, instead of downloading and saving every icon. 
It can be slower to start or not at all (i.e. if the CDN is down the icons will not load) but that's pretty rare.

Reply:
👍

Reply:
Okay wait, you are right but @chaitali and I can't figure out why this project wont let me use fontawesome? Like it just wont import it? Maybe because we have some sort of font dependency?

Reply:
I could try importing like this instead? https://fontawesome.com/how-to-use/on-the-web/using-with/react